### DISCUSSION TITLE
第一次在本地搭建alist就遇到问题，请各位路过的大神帮忙指点一下，谢谢

### DISCUSSION THREAD

**Discussion by jyotan0817** [AlistGo/alist#5642] [Q&A]
搭建平台：win11 家庭版
搭建过程：CMD(admin模式）中安装alist 出现错误，程序被终止。
![db26b44b4e42d5dc3b4490bea1e1e37](https://github.com/alist-org/alist/assets/52436686/01742595-da3b-429c-94a1-cc3e0f8981a8)

检查过网络、防火墙、防护软件都已经关闭，问题依旧存在，请大神帮忙指点谢谢

**Comment by kaixinxiaowangzi666** [comment]
打开看看教程https://www.bilibili.com/video/BV1Te4y1G7r1/?spm_id_from=333.999.0.0&vd_source=c17b0a1a925f2ed8b095c047cba77c95

**Reply by jyotan0817** [reply]
谢谢，可以用了

**Comment by xhofe** [comment]
你的截图中已经在成功运行了，并没有你说的“安装alist 出现错误，程序被终止”

**Reply by jyotan0817** [reply]
可以了，已经成功登录

**Comment by Colorhhy** [comment]
请问怎么解决的 我的一直有这个提示

**Reply by kaixinxiaowangzi666** [reply]
https://www.bilibili.com/video/BV1Te4y1G7r1/?spm_id_from=333.999.0.0&vd_source=c17b0a1a925f2ed8b095c047cba77c95
